addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24026,7 +24025,13 @@ msgctxt "#40038"
 msgid "The default version of a movie with multiple versions cannot be added to another movie. Please move or remove the other versions first."
 msgstr ""
 
-#empty strings with id 40039 to 40199
+#. Manage video versions context menu item
```

Comment:
should be `manage video extras context menu item`

Reply:
yes, done